tensilelite/Tensile/Components/LSU.py

```diff
@@ -227,7 +227,7 @@ def writeReadReduction(self, writer, kernel):
             dataPerWave = numAccVgpr * kernel["WavefrontSize"] * 4
             ldsStride   = dataPerWave * numWaves
 
-            addr = writer.vgprPool.checkOut(1,"addr")
+            addr = writer.vgprPool.checkOut(2,"addr")
```

Comment:
It is not a good idea to always increase the number of vgpr register here.
At least, we should be able check if max LDS size is larger than 65536 or not.

Ideally, we can calculate all offset values used here in advance and check the max offset value.
Then, we can decide how many addr registers are actually needed here.